### ISSUE TITLE
The whole lodash library is importing in Pagination control

### ISSUE THREAD

**Issue by lirik30** [pnp/sp-dev-fx-controls-react#1018]
#### Category
[x] Enhancement

[ ] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [ 3.2.1 ]

#### Expected / Desired Behavior / Question
In [`Pagination.tsx`](https://github.com/pnp/sp-dev-fx-controls-react/blob/master/src/controls/pagination/Pagination.tsx) only `isEqual` is importing from `lodash`. 

#### Observed Behavior
The whole `lodash` library are importing.

#### Steps to Reproduce
1. I'm facing that issue while using `Pagination` control in my webpart, so to reproduce, create simple spfx webpart and import that control.
```typescript
import { Pagination } from '@pnp/spfx-controls-react/lib/Pagination';
```
2. Install and configure [webpack-bundle-analyzer](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/toolchain/optimize-builds-for-production#verify-the-contents-of-your-bundle)
3. Check the _.stats.html file. That one is mine 
![wba_wrong](https://user-images.githubusercontent.com/20610579/134519175-56a94d3a-2797-4c86-9b92-11f7240f594d.png)
4. To fix, just cherry-pick `isEqual` from `lodash` in [`Paginationy.tsx`](https://github.com/pnp/sp-dev-fx-controls-react/blob/master/src/controls/pagination/Pagination.tsx):
```typescript
import isEqual from "lodash/isEqual";
```
5. As a result:
![wba_fixed](https://user-images.githubusercontent.com/20610579/134520328-70ec49d6-3772-4700-8f2a-5960f1c05968.png)

**Comment by joelfmrodrigues** [comment]
@lirik30 thanks for reporting the issue.
I have marked it as "Good First Issue" and will fix it if no one else sends a PR (you are welcome to send a PR if interested)

**Comment by joelfmrodrigues** [comment]
@lirik30 thanks for the PR, this will be available shortly as part of the automated beta release and included in the next main release

**Comment by AJIXuMuK** [comment]
This has been fixed as part of 3.4.0
